package.json

```diff
@@ -3,23 +3,32 @@
   "private": true,
   "license": "MIT",
   "scripts": {
-    "dev": "next dev",
+    "dev": "(local-ssl-proxy --source 4000 --target 3000) & next dev",
```

Comment:
Ahh... I think I'll use [`npm-run-all`](https://www.npmjs.com/package/npm-run-all) then since it will allow us to have only one place to define tasks.

Reply:
Okay. Didn't know about this lib. I'll take a look into using it for my PR, seems like a much cleaner solution